README.md

```diff
@@ -293,5 +349,9 @@ $ knife clc template list --datacenter ca1
 5. Create a new Pull Request
 6. Specs and Code Style checks should pass before Code Review.
 
+## FAQs
```

Comment:
Let's not add FAQ section for now. There's nothing to put there (at least for now).